content/articles/getting-started-with-burpsuite/index.md

```diff
@@ -0,0 +1,139 @@
+Burpsuite is one of the most popular web application security tools. Burp Suite is an intercepting proxy tool that allows us to intercept, analyze and modify requests coming from our browsers before they are sent to the remote server. With burp suite we can intercept HTTP messages, modify the header and body of a message, and manually test for vulnerabilities.
+
+Burpsuite is one of those important tools found in the arsenal of web application penetration testers, web application security researchers, CTF players, as well as bug bounty hunters. 
+
+![Proxy](/engineering-education/getting-started-with-burpsuite/illustration.png)
+
+Why Burpsuite?
+
+Burpsuite is not only a proxy tool. It is a master tool that can be used to perform a series of tasks like:
+
+- Web spidering 
+- Automated and Manual testing of web applications.
+- Web application analysis
+- Finding vulnerabilities 
+
+Another plus is its inbuilt chromium-browser.
+
+Burpsuite is available as a free community edition and a professional edition which costs about $3999 a year.
+
+In this article, we'll be going through the basic usage of burp suite.
+
+**Disclaimer:** This article is for educational purposes only.
+
+### Prerequisites
+
+- Linux or Windows OS
+- Mozilla Firefox Browser
```

Comment:
When you want to add the "Prerequisites," section, always add something like "To follow along with this tutorial, you are required to have the following:"

Delete the entire section. The user can use any browser.